tests/network/openvpn_client.pm

```diff
@@ -37,7 +38,12 @@ sub run {
     mutex_wait 'OPENVPN_STATIC_KEY';
 
     # Download the client config
-    assert_script_run("curl -o static.conf " . data_url("openvpn/static_client.conf"));
+    if (is_sle('>=15-sp4')) {
+      assert_script_run("curl -o static.conf " . data_url("openvpn/static_client.conf"));
```

Comment:
just an FYI, no need to change it or rerun vrs, but you could make this simpler with:

```
my $conf_file = is_sle('>=15-sp4') ? 'static_client.conf' : 'static_client_legacy.conf';
assert_script_run("curl -o static.conf " . data_url("openvpn/$conf_file"));
```